Source/Plugins/Core/com.equella.base/src/com/tle/common/searching/Search.java

```diff
@@ -93,4 +93,17 @@ public SortField getSortField(boolean reverseTheDefault) {
    * @return True if server time zone is used or otherwise false.
    */
   boolean useServerTimeZone();
+
+  /**
+   * The background of creating this method is to support the Lucene query built in New Advanced
```

Comment:
Rather than just jumping straight into the background, be good to also have the description of the method still.

```suggestion
   * Returns an optional additional query which is in raw text Lucene query syntax, available to be merged with other
   * query parameters.
   *
   * <p>The background of creating this method is to support the Lucene query built in New Advanced
```